gbmplus/api/orders.py

```diff
@@ -71,6 +71,41 @@ def submitOrder(self, legacy_contract_id, duration, order):
 
         return self._session.post(metadata, resource, payload)
     
+
+    def submitOrderToStrategy(self, strategy_name, issuer, quantity, order_type, trading_type, instrument_type, price=None, duration=1):
+        """
+        **Submit an order to a specific strategy**
+        https://homebroker-api.gbm.com/GBMP/api/Operation/RegisterCapitalOrder
+
+        - strategy_name (string): (required)
+        - issuer (string): (required)
+        - quantity (int): (required)
+        - order_type (enum): (required)
+        - trading_type (enum): (required)
+        - instrument_type (required)
+        - price (float): (optional)       
+        - duration (int): (optional)
+        """        
+                
+        metadata = {
+            'tags': ['order', 'generate order'],
+            'operation': 'submitOrder'
+        }
+                
+        resource = "https://homebroker-api.gbm.com/GBMP/api/Operation/RegisterCapitalOrder"
+        
+        legacy_contract_id = gbm_metadata.accounts_dict.get(strategy_name).get("legacy_contract_id")
```

Comment:
Here we could have a bug. If `gbm_metadata.accounts_dict.get(strategy_name) == None` then `None.get("legacy_contract_id")` will raise an Exception which could take a time the user to fix. Raise an early Exception may be nice and informative to the user:
```suggestion
        if strategy_name not in gbm_metadata.accounts_dict:
            raise Exception(f"strategy_name '{strategy_name}' is unknown")
            
        legacy_contract_id = gbm_metadata.accounts_dict.get(strategy_name).get("legacy_contract_id")
```

or an easiest fix will be:
```suggestion
        legacy_contract_id = gbm_metadata.accounts_dict.get(strategy_name, {}).get("legacy_contract_id")
```